tcf_website/views/team_info.json

```diff
@@ -129,6 +129,12 @@
       "github": "akh4rf"
     },
     {
+      "name": "Vinya Ponugoti",
+      "role": "Developer",
+      "class": "2024",
+      "img_filename": "ENG_Vinya_Ponugoti.jpg",
+      "github": "vinyaponugoti"
+    },
```

Comment:
Seems like you're missing an opening `{` here - probably why your CI build failed.